.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,36 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
```

Comment:
please look at https://github.com/dlt-hub/verified-sources/blob/master/.github/ISSUE_TEMPLATE/bug-report.md

we should get more context on a bug (not always applicable) ie.
- data source type (verified source? API? just data)
- destination (bigquery? etc.)
- runtime env. so we know this is production or local laptop or Colab notebook

Reply:
ohhh I'd also ask them to go to slack before and report there. I think we should have people check out tech help on slack before reporting bugs